docs/oscar-to-carlos-bootstrap-conversion.md

```
@@ -0,0 +1,284 @@
+    <div class="text-muted small">Long Title / Patient Name / Context Info</div>
+</div>
+```
+
+---
+
+## JSP-Specific Notes
+
+When the converted file is a `.jsp` (not a static `.html`):
+
+1. **Replace the `<head>` block** with a single JSP include:
+   ```jsp
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)